pkg/splunk/enterprise/monitoringconsole_test.go

```diff
@@ -679,3 +691,491 @@ func TestMonitoringConsoleGetAppsListForAWSS3ClientShouldFail(t *testing.T) {
 		t.Errorf("GetAppsList should return an empty response since we have empty objects in MockAWSS3Client")
 	}
 }
+
+func TestMonitoringConnsoleWithReadyState(t *testing.T) {
+
+	mclient := &spltest.MockHTTPClient{}
+	type Entry1 struct {
+		Content splclient.ClusterMasterInfo `json:"content"`
+	}
```

Comment:
why do we need clustermaster info here?

Reply:
removed

Reply:
I believe you plan to remove this? correct?